pages/common/git-annex.md

```diff
@@ -0,0 +1,28 @@
+# git annex
+
+> Manage files with git, without checking their contents in.
+> When  a  file  is annexed, its content is moved into a key-value store, and a symlink is made that points to the content. More information: <https://git-annex.branchable.com/>.
```

Comment:
Perhaps this will read better?
```suggestion
> Manage files with git, without checking their contents in.
> When a file is annexed, its content is moved into a key-value store, and a symlink is made that points to the content.
> More information: <https://git-annex.branchable.com/>.
```